src/blocks/battery.rs

```diff
@@ -10,7 +10,7 @@
 //! `device` | The device in `/sys/class/power_supply/` to read from. When using UPower, this can also be `"DisplayDevice"`. Regular expressions can be used. | Any battery device
 //! `driver` | One of `"sysfs"`, `"apc_ups"`, or `"upower"` | `"sysfs"`
 //! `interval` | Update interval, in seconds. Only relevant for `driver = "sysfs"` \|\| "apc_ups"`. | `10`
-//! `format` | A string to customise the output of this block. See below for available placeholders. | <code>"$percentage&vert;"</code>
+//! `format` | A string to customise the output of this block. See below for available placeholders. | <code>" $icon $percentage&vert; "</code>
 //! `full_format` | Same as `format` but for when the battery is full | `""`
 //! `empty_format` | Same as `format` but for when the battery is empty | `""`
 //! `hide_missing` | Completely hide this block if the battery cannot be found. | `false`
```

Comment:
Not sure about `hide_missing`, but `hide_full` seems redundant now.

Reply:
`hide_missing` needs to be present because `$icon` value is still provided and the block would be rendered. I'll drop hide_full and noticed another bug, `$icon` needs be set in default values for `full_format` and `empty_format`

Reply:
Regarding `hide_missing`. What do you think of updating both battery and net blocks to similar configuration - `format_missing` instead of `hide_missing`? This would address https://github.com/greshake/i3status-rust/pull/1618#issuecomment-1242766382

Reply:
I think it's a good idea.

Reply:
Added in https://github.com/greshake/i3status-rust/pull/1618/commits/56828b57776da45a19afcd550f796642c09d7f8b